news.js

```diff
@@ -0,0 +1,19 @@
+  function fetchData() {
+
+    fetch("http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/uk-news?show-fields=all").then(response => {
+      return response.json();
+    }).then(data => {
+      console.log(data);
```

Comment:
I think this line is no longer necessary - I'm guessing you used it earlier to check the fetch request was working. :)